test/resources/case38_operator_install/operator-policy-all-defaults.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: policy.open-cluster-management.io/v1beta1
+kind: OperatorPolicy
+metadata:
+  name: oppol-all-defaults
+  annotations:
+    policy.open-cluster-management.io/parent-policy-compliance-db-id: "124"
+    policy.open-cluster-management.io/policy-compliance-db-id: "64"
+  ownerReferences:
+  - apiVersion: policy.open-cluster-management.io/v1
+    kind: Policy
+    name: parent-policy
+    uid: 12345678-90ab-cdef-1234-567890abcdef # must be replaced before creation
+spec:
+  remediationAction: enforce
+  severity: medium
+  complianceType: musthave
+  subscription:
+    name: project-quay
+    namespace: operator-policy-testns
+    installPlanApproval: Manual
```

Comment:
This is separate from this PR, but I think that the controller will approve the InstallPlan for this operator... Is that confusing, because the user specified "Manual"? What do you think?

Reply:
Based on our conversation about this, @JustinKuli will create a separate issue to address this.